daos/mainnet/daofund.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "DAOfund",
+    "Avatar": "0xe8e1D0f1783b22ff409DeB14Fa5ACC9ccE2a3D15",
+    "DAOToken": "0x3e8Bcf4ce0A3F54D0Bd1B0799f7abfc4C5B9FBae",
+    "Reputation": "0x462dE5f48155cFB8ACd76564930d737cA55e35aA",
+    "Controller": "0x8298C1A556C2847aE6Ae937b456C9B480D78e75F",
+    "Schemes": {},
+  "arcVersion": "0.0.1-rc.22"
```

Comment:
are we supporting rc22 ?

Reply:
Yes seems so from the mappings.json file